File: app/src/main/java/com/example/myapplication/machinecoding/wallet/java/dao/IWalletDao.java
package com.example.myapplication.machinecoding.wallet.java.dao;

import com.example.myapplication.machinecoding.wallet.java.models.Account;

import java.util.TreeMap;

public interface IWalletDao {
    int createAccount(String name, double amount);

    TreeMap<Integer, Account> getAccounts();

    Account getAccount(int id);

    void updateAccount(int id, Account user);
}

File: app/src/main/java/com/example/myapplication/machinecoding/wallet/kotlin/dao/IWalletDao.kt
package com.example.myapplication.machinecoding.wallet.kotlin.dao

import com.example.myapplication.machinecoding.wallet.kotlin.models.Account
import java.util.TreeMap;


interface IWalletDao {
    fun createAccount(name: String?, amount: Double): Int
    val accounts: TreeMap<Int?, Account?>?

    fun getAccount(id: Int): Account?
    fun updateAccount(id: Int, user: Account?)
}

